### DISCUSSION TITLE
Importing CSS files as text like HTML files

### DISCUSSION THREAD

**Discussion by Deel96** [parcel-bundler/parcel#4723] [General]
I try to get the content of a CSS file.
For HTML files this works: 
`import html from "./home.html";`

and in html I get the content of the file as a string.

But with CSS I get an empty object 
`import css from "./home.css"`
![grafik](https://user-images.githubusercontent.com/37439779/84256190-df75f780-ab13-11ea-985e-1e93c67fb951.png)

Is there a way to import it in the same way?
I use typescript and have a html.s.ts and a css.d.ts file with following content:
```
declare module '*.html' {
    const value: string;
    export default value
}
declare module '*.css' {
    const value: string;
    export default value
}

```

My tsconfig:
```
{
  "compilerOptions": {
    "noImplicitAny": false,
    "noEmitOnError": true,
    "removeComments": false,
    "sourceMap": true,
    "target": "es6",
    "moduleResolution": "Node"
  },
  "exclude": [
    "node_modules",
    "wwwroot"
  ]
}

```

I use parcel-bundler 1.12.4

**Comment by Deel96** [comment]
Is there no way to do this?

**Comment by mischnic** [comment]
Only with Parcel 2: https://v2.parceljs.org/configuration/plugin-configuration/#predefined-(offical)-named-pipelines
